src/main/webapp/js/pages/index.js

```diff
@@ -34,10 +34,41 @@ document.querySelector('.course-list').addEventListener('click', () => {
       course => {sections2[course] = courseInfo[course].sections[1]});
   const schedules = [sections1, sections2];
 
-  // TODO(ramyabuva): add pagination for multiple schedules
-  selected.forEach(course => {
-    Calendar.addCourse(courseInfo[course], schedules[0][course]);
-  });
+  function addScheduleToCalendar(schedule){
+    Calendar.clear();
+    selected.forEach(course => {
+      Calendar.addCourse(courseInfo[course], schedule[course]);
+    });
+  }
+
+  addScheduleToCalendar(schedules[0]);
```

Comment:
why are you selecting only the first element? if it's just for testing (until we hook up other parts of the app), please make a comment explaining.